velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -348,6 +387,8 @@ void MinMaxByAggregationTestBase::SetUp() {
   ASSERT_EQ(dataVectorsByType_.size(), kSupportedTypes.size());
   rowVectors_ = makeVectors(rowType_, 5, 10);
   createDuckDbTable(rowVectors_);
+  rowVectorsWithHugeInt_ = makeVectors(rowTypeWithHugeInt_, 5, 10);
+  createDuckDbTable(rowVectorsWithHugeInt_);
```

Comment:
MinMaxBy tests use this `std::vector<RowVectorPtr> rowVectors_;` member as input for multiple test cases. This row vector has all data columns (except HUGEINT). If I append HUGEINT type to this rowVector_, then I see the `HUGEINT failure with DWRF`. To avoid this issue and as well test HUGEINT compatibility with other data types, I created another RowVector, rowVectorsWithHugeInt and use in tests wherever `rowVectors` are used.

Thanks for pointing out, yes, I was overwriting the table. This has not caused issue so far because the test seems to generate same set of values for each RowVector for with or without hugeint columns:

```
ROW 0 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 0 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 1 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 1 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 2 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 2 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 3 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 3 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 4 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 4 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 5 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 5 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 6 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 6 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 7 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 7 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 8 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 8 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 
ROW 9 true null -27624 null 5569314186296520615 0.12518276274204254 0.6463130115983018 i -1799777366 2015-10-13T00:07:54.000276629 

```
Even without my changes, the data in each row is same and it is not an effective way to test min_by and max_by function. I will look more into this.

Reply:
I made a change to keep rowVectors_ and rowVectorsWithHugeInt_ consistent. I copy rowVectorsWithHugeInt_ into rowVectors_ by excluding HUGEINT column.

Reply:
> then I see the `HUGEINT failure with DWRF`

Can't we add a check to exclude HUGEINT for DWRF? The current setup is confusing.